src/hb-aat-layout.cc

```diff
@@ -46,11 +46,16 @@
  * @short_description: Apple Advanced Typography Layout
  * @include: hb-aat.h
  *
- * Functions for querying OpenType Layout features in the font face.
+ * Functions for querying OpenType Layout features in the font face. 
```

Comment:
This line looks like it was copied from somewhere else, since this file has AAT not OpenType functions.